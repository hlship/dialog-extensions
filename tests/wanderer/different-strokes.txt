%% Same quip may have different text based on who is being talked to
%% This also tests pronouns

talk to the barmaid
ask the barmaid about rumors 
ask about garrick
talk to the wanderer 
ask him about the rumors
ask knows
%% It should be the miniature
ask about it
