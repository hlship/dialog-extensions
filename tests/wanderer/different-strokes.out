

It is a long and riotous evening, full of unlikely stories and tall tales. But
now, most of the patrons have gone away to their rooms to sleep, or have passed
out before the fire. Even the two black bitch pups are curled on the hearth-
stone, snuffling through tiny wet noses, and pawing the air in sleep. Now is the
time to find out whether the rumors that brought you to this neighborhood are
true.

The Inn
The light is low and red. Above the fireplace hangs a map of this part of the
Empire, and on the adjacent wall is the crimson and silver banner of Richard the
Rose. The barmaid works her way from table to table, clearing away dishes and
mugs.

A wanderer in grubby trousers slouches at the seat nearest the fire.

> talk to the barmaid
You say hello to the Bar Maid.

You could ask whether the rumors tell truly or ask where the barmaid comes from.

> ask the barmaid about rumors 
"Where I come from, over the black hills there, they say that men this side of
the mountain live as old as Methuselah," you remark. "They say the secret of
eternal life is here."

"Oh, do they?" she says, sweeping crumbs of cheese and crusty bread into her
hand. "The oldest codger around these parts is old Garrick, and I would't put
him beyond his four-score and ten."

You could ask where Garrick lives, ask whether she's heard the stories, or ask
where the barmaid comes from.

> ask about garrick
"Where does this old Garrick live?" you ask, trying not to seem too eager.

"Down at the pig farm," she says. "The turn off is just before you come to the
river, on the main road west. You can find it by the smell."

You could ask whether she's heard the stories or ask where the barmaid comes
from.

> talk to the wanderer 
You say hello to the Wanderer.

> ask him about the rumors
"Where I come from, over the black hills there, they say that men this side of
the mountain live as old as Methuselah," you remark. "They say the secret of
eternal life is here."

"That's true enough," says the Wanderer. "Though there are plenty around here
that will deny it."

You could ask what the wanderer knows.

> ask knows
"What do you know about those who live forever?" you ask.

"I met last month a man that used to be a friend of my grandfather. My
grandfather's long dead and in the ground, and this man looks younger than
myself. I have a miniature painting of them two together, and I tell you he has
not aged a moment."

You could ask whether you may see the miniature.

> ask about it
"May I see this miniature? I'm curious to see the man who has lived without
aging."

"Why not?" He searches through the pockets of his coat, then the pockets of his
trousers, and finally--looking surprised and much relieved--finds what he was
looking for tucked away in his boot. "Here is is: have a look."

And he extends for view an old-fashioned locket: painted on an ivory rectangle
are the images of two young men. They are painted so small it would be hard to
guarantee that you would recognize them again.

> 
